### ISSUE TITLE
ssl.VerifyMode and other Enums not recognized

### ISSUE THREAD

**Issue by cipherboy** [PyCQA/pylint#3691]
### Steps to reproduce
1. Recreate this file:
```python
#!/usr/bin/python3

"""
Proof of Concept: ssl.VerifyMode isn't understood by pylint
"""

import ssl

print(ssl.VerifyMode)
```
2. Run `pylint` against it:
```bash
$ ~/.local/bin/pylint ./example.py
************* Module example
example.py:9:6: E1101: Module 'ssl' has no 'VerifyMode' member (no-member)

----------------------------------------------------------------------
Your code has been rated at -15.00/10 (previous run: -15.00/10, +0.00)

$ ~/.local/bin/pylint --version
pylint 2.5.3
astroid 2.4.2
Python 3.8.3 (default, May 15 2020, 00:00:00) 
[GCC 10.1.1 20200507 (Red Hat 10.1.1-1)]
$ pylint-3 ./example.py
************* Module example
example.py:9:6: E1101: Module 'ssl' has no 'VerifyMode' member (no-member)

----------------------------------------------------------------------
Your code has been rated at -15.00/10 (previous run: -15.00/10, +0.00)
$ pylint-3 --version
pylint 2.4.4
astroid 2.3.3
Python 3.8.3 (default, May 15 2020, 00:00:00) 
[GCC 10.1.1 20200507 (Red Hat 10.1.1-1)]
```
3. Notice that the member does exist:
```bash
$ python3 ./example.py
<enum 'VerifyMode'>
$ python3 --version
Python 3.8.3
```

### Current behavior

[`ssl.VerifyMode`](https://docs.python.org/3/library/ssl.html#ssl.VerifyMode) doesn't get detected

### Expected behavior

`ssl.VerifyMode` should be detected.

### pylint --version output

```
pylint 2.5.3
astroid 2.4.2
Python 3.8.3 (default, May 15 2020, 00:00:00) 
[GCC 10.1.1 20200507 (Red Hat 10.1.1-1)]
```

and

```
pylint 2.4.4
astroid 2.3.3
Python 3.8.3 (default, May 15 2020, 00:00:00) 
[GCC 10.1.1 20200507 (Red Hat 10.1.1-1)]
```

----

I believe this has to do with how it is constructed in the [module](https://github.com/python/cpython/blob/master/Lib/ssl.py#L122-L150):

```python
_IntEnum._convert_(
    '_SSLMethod', __name__,
    lambda name: name.startswith('PROTOCOL_') and name != 'PROTOCOL_SSLv23',
    source=_ssl)

_IntFlag._convert_(
    'Options', __name__,
    lambda name: name.startswith('OP_'),
    source=_ssl)

_IntEnum._convert_(
    'AlertDescription', __name__,
    lambda name: name.startswith('ALERT_DESCRIPTION_'),
    source=_ssl)

_IntEnum._convert_(
    'SSLErrorNumber', __name__,
    lambda name: name.startswith('SSL_ERROR_'),
    source=_ssl)

_IntFlag._convert_(
    'VerifyFlags', __name__,
    lambda name: name.startswith('VERIFY_'),
    source=_ssl)

_IntEnum._convert_(
    'VerifyMode', __name__,
    lambda name: name.startswith('CERT_'),
    source=_ssl)
```

**Comment by brycepg** [comment]
Thanks for the report. I can verify this bug. If someone would like to have a go at fixing this, I would recommend populating these variables in `astroid` using a brain tip. See http://pylint.pycqa.org/projects/astroid/en/latest/extending.html#ast-inference-tip-transforms and https://github.com/PyCQA/astroid/tree/master/astroid/brain
